pkg/devices/led.go

```diff
@@ -28,23 +23,29 @@ var LED = sdk.DeviceHandler{
 // ledRead is the read handler for the emulated LED device(s). It
 // returns the state and color values for the device.
 func ledRead(device *sdk.Device) ([]*sdk.Reading, error) {
-	if state == "" {
-		state = stateOff
-	}
-	if color == "" {
-		color = "000000"
-	}
+	var state, color string
 
 	dState, ok := deviceState[device.ID()]
 
 	if ok {
-		if _, ok := dState[state]; ok {
-			state = dState[state].(string)
+		if _, ok := dState[color]; ok {
+			color = dState[state].(string)
```

Comment:
I'm pretty sure the key into the dState map should be the string `"color"` and `"state"` instead of the value in the var

Also, it looks like there are mixed state/color refs

```go
if _, ok := dState["color"]; ok {
    color = dState["color"].(string)
}

if _, ok := dState["state"]; ok {
    state = dState["state"].(string)
}
```

Reply:
gah that was sloppy, my b